lib/fizzy/execute.cpp

```diff
@@ -356,6 +362,62 @@ T fnearest(T value) noexcept
         return t;
 }
 
+
+template <typename T>
+T fabs(T) noexcept = delete;
+
+template <>
+float fabs(float input) noexcept
+{
+    auto result = bit_cast<unsigned int>(input);
+    result &= ABS_MASK;
+    return bit_cast<float>(result);
+}
+
+template <>
+double fabs(double input) noexcept
+{
+    auto result = bit_cast<uint64_t>(input);
+    result &= ABS_MASK64;
+    return bit_cast<double>(result);
+}
+
+template <typename T>
+T neg(T input) noexcept
+{
+    if constexpr (std::is_same_v<T, float>)
+    {
+        auto result = bit_cast<unsigned int>(input);
+        result ^= SIGN_MASK;
+        return bit_cast<T>(result);
+    }
+    else
+    {
+        auto result = bit_cast<uint64_t>(input);
+        result ^= SIGN_MASK64;
+        return bit_cast<T>(result);
+    }
+}
+
+template <typename T>
+T copysign(T a, T b) noexcept
```

Comment:
I noticed that some conversion functions may use `fabs()`, `copysign()` and others. So please move these functions before `fceil()` in this file.